source/includes/markdown/_13-platform-ui.html.md

```diff
@@ -278,11 +271,190 @@ Related References:
+5. Once the example app is installed, create a task in your project. In the task's **{Example}** custom field, go to **{Add Example Resource}** > **{Open form}** to see examples of customizable inputs. Click **Submit**.
+<br>
+<br>
+<img src="../images/example-app-form.gif" />
+<br>
+<br>
+6. View the newly-generated widget on your task. You can begin editing `index.js` to modify the contents of the widget. Note that you'll need to restart the local server and reload the page to see your changes.
+<br>
+<br>
+<img src="../images/example-app-widget.png" />
+<br>
+<br>
+That's it! At this point, feel free to keep exploring how changes in the server affects data in the task's widget. Once you're ready to define an app, [click here to create your own app](/docs/configurations) with App Components.
+
+<hr class="full-line">
+
+# Publishing an App
+
+<span class="beta-indicator">BETA</span> - For access, please see [Overview of App Components](/docs/overview-of-app-components)
+
+Apps built on App Components are manually reviewed before they are accessible within Asana. To ensure a smooth review process and user experience, here are some guidelines you can follow.
+
+<hr>
+
+## General Guidelines
+
+<span class="beta-indicator">BETA</span> - For access, please see [Overview of App Components](/docs/overview-of-app-components)
+
+When submitting an [app configuration](/docs/configurations), you should:
+
+* Add necessary images (e.g., feature images, icon, logo, etc.)
+* Add support links (e.g., feedback link, external support URL)
+* Proofread marketing-related text (e.g., description, extended description, features)
+* Make sure button text has 3-4 words or fewer and start with a verb
+* Use consistent language for similar concepts where applicable
+* Use sentence case by default and capitalize proper nouns
+
+When testing your application, you should:
+
+* Try to "break" your forms (e.g., test watched fields, limit invalid submissions, test typeahead fetches, etc.)
+* Test and proof-read any custom error messages
+* Test the auth flow from both the web browser and [desktop app](https://asana.com/download)
+* Test app actions with a variety of trigger combinations
+
+<hr>
+
+## Security
+
+<span class="beta-indicator">BETA</span> - For access, please see [Overview of App Components](/docs/overview-of-app-components)
+
+When handling requests from Asana, an App Components app should:
+ 
+ - Reject requests with missing or incorrect signatures.
+ - Reject requests with an "expires" time in the past.
+    - **Note**: Make sure to use the correct units for this. "expires" is in milliseconds. If you compare the expiration time to a timestamp in seconds, it will always look like the request expires thousands of years in the future.
+
+If an app uses OAuth for authentication, the app should:
+ 
+ - Prevent OAuth CSRF attacks. This is often done using a one-time CSRF token in the "state" parameter. This can also be done using PKCE instead, if it's supported.
```

Comment:
Perhaps work linking to the relevant OAuth doc sections here?

Reply:
Good call. Will add!